File: scratch.js
const https = require('https');
https.get('https://attrangs.co.kr/shop/search.php?search_text=ps4533', { headers: { 'User-Agent': 'Mozilla/5.0' } }, (resp) => {
  let data = '';
  resp.on('data', (chunk) => { data += chunk; });
  resp.on('end', () => {
    const fs = require('fs');
    fs.writeFileSync('attrangs_html.txt', data);
    console.log('done');
  });
});

File: scratch.py
import urllib.request
import re

url = 'https://attrangs.co.kr/shop/search.php?search_text=ps4533'
req = urllib.request.Request(url, headers={'User-Agent': 'Mozilla/5.0'})
try:
    html = urllib.request.urlopen(req).read().decode('utf-8')
    img_matches = re.findall(r'<img[^>]+src=[\"\']([^\"\']+)[\"\']', html)
    for img in img_matches:
        if 'ps4533' in img or 'data/item' in img or 'thumb' in img:
            print(img)
except Exception as e:
    print('Error:', e)
